apps/src/componentLibrary/dropdown/checkboxDropdown/CheckboxDropdown.tsx

```diff
@@ -62,6 +64,7 @@ export interface CheckboxDropdownProps {
 const CheckboxDropdown: React.FunctionComponent<CheckboxDropdownProps> = ({
   name,
   labelText,
+  labelType,
```

Comment:
Thanks for spotting! It's not required here since `_CustomDropdown` has it's own default value, but I agree at makes more sense to also define it here cause we do the same for other props that has default values